src/js/background.js

```diff
@@ -592,11 +592,15 @@ Badger.prototype = {
     if (disabledSites && disabledSites.length > 0) {
       for (var i = 0; i < disabledSites.length; i++) {
         var site = disabledSites[i];
+
         if (site.startsWith("*")) {
-          if (window.getBaseDomain(site) === window.getBaseDomain(origin)) {
+          var wildcard = site.slice(2); // remove "*."
```

Comment:
Should this be `site.slice(1)` instead? If we remove two characters, `*.example.com` will match `example.com`.

Also, the dot may not be there, which is fine if we remove one character (but not quite right if we remove two).

Reply:
Would be good to add a test for this scenario.